src/ChatWindow.js

```diff
@@ -71,6 +71,7 @@ class ChatWindow extends Component {
   Update the state as the user is typing into the input box
   */
   handleChangeInUserRequestToAlexa(event) {
+    event.preventDefault();
```

Comment:
Follow the same naming convention for event through out the code i.e., either 'e' or 'event'.

Reply:
Done. Renamed to event.